Fortran/gfortran/regression/DisabledFiles.cmake

```diff
@@ -2941,22 +3012,6 @@ file(GLOB FAILING_FILES CONFIGURE_DEPENDS
   openacc-define-3.f90
   openmp-define-3.f90
 
-  # Tests looking for runtime errors (e.g., bound checks). Correctly
-  # caught by flang runtime when it is used for array assignments.
-  all_bounds_1.f90
-  bounds_check_12.f90
-  bounds_check_array_ctor_4.f90
-  bounds_check_fail_3.f90
-  maxloc_bounds_1.f90
-  maxloc_bounds_2.f90
-  maxloc_bounds_3.f90
-  maxloc_bounds_4.f90
-  maxloc_bounds_5.f90
-  maxloc_bounds_7.f90
-  maxloc_bounds_8.f90
-  pack_bounds_1.f90
-  spread_bounds_1.f90
```

Comment:
Ah, I misunderstood your comment in the original.

Interestingly, when I enable the test, they fail anyway despite correctly generating a runtime error. This seems to be a bug in the way I check for failures. I will look into this. 

Thanks for the catch!

Reply:
> Interestingly, when I enable the test, they fail anyway despite correctly generating a runtime error. This seems to be a bug in the way I check for failures. I will look into this.

This occurred because calling abort from the test instantly caused the test to be reported as a failure without running the custom verifier which checks for a non-zero return code. 

The easiest way to fix this is to use LLVM's `not` utility. But we cannot rely on the utility being available since we may not have LLVM's build directory when running the test suite and LLVM could have been configured with `-DLLVM_INSTALL_UTILS=OFF`.